datasets/leidingeninfrastructuur/dataset.json

```
@@ -14,38 +14,44 @@
   "accrualPeriodicity": "dagelijks",
   "spatialDescription": "Gemeente Amsterdam",
   "theme": [
-    "leidingeninfrastructuur",
+    "ondergrondse infrastructuur",
     "Ruimte en Topografie"
   ],
-  "owner": "Gemeente Amsterdam, Ruimte & Economie",
+  "owner": "Gemeente Amsterdam",
   "auth": "OPENBAAR",
-  "authorizationGrantor": "datateam omgevingswet",
+  "authorizationGrantor": "datateamstedelijkeonwikkelingenbeheer@amsterdam.nl",
   "keywords": [
     "leidingen",
-    "infrastuctuur"
+    "infrastuctuur",
+    "klic",
+    "waternet",
+    "rioolnetwerk",
+    "liander"
   ],
   "hasBeginning": "2021-11-01T00:00:00+01:00",
   "crs": "EPSG:28992",
   "tables": [
     {
-      "id": "liggingLijnTotaal",
-      "$ref": "liggingLijnTotaal/v1.2.0",
+      "id": "kadasterKlicKabelsEnLeidingen",
+      "shortname": "klicKabelsLeidingen",
```

Sorry, ik dacht dat ik naar velden zat te kijken ipv. tabellen. Dit probleem is alleen op te lossen door de id's korter te maken, of (makkelijker) door de hele dataset te hernoemen tot bijv. ondergrondseinfra?

```suggestion
      "shortname": "klicKabelsLeidingen",
```